packages/ember-data/lib/system/debug/debug-adapter.js

```diff
@@ -42,7 +42,11 @@ export default Ember.DataAdapter.extend({
     return columns;
   },
 
-  getRecords: function(modelName) {
+  getRecords: function(modelNameOrFactory) {
+    // TODO: Ask Teddy what we should do here.
+    // Ideally this should always get passed a string.
+
+    var modelName = typeof modelNameOrFactory === 'string' ? modelNameOrFactory : modelNameOrFactory.modelName;
```

Comment:
I don't think this gives user's an action as it's in the debug adapter for the Ember Inspector. We try to avoid deprecations for code that isn't the user's.